pkg/sensors/exec/procevents/proc_reader.go

```diff
@@ -500,13 +578,13 @@ func listRunningProcs(procPath string) ([]procs, error) {
 
 		execPath, err := os.Readlink(filepath.Join(procPath, d.Name(), "exe"))
 		if err != nil {
-			logger.GetLogger().WithError(err).WithField("process", d.Name()).Warnf("reading process exe error")
+			execPath = strings.TrimSuffix(string(cmdline), "\n")
```

Comment:
hm, how this became non error case?

Reply:
In the case of kernel threads, `/proc/<pid>/exe` does not make sense. So in that case, I get the `execPath` from the `kthread` name. But still, for the non-kernel threads, we should report this error. I will update it to something like:
```
if kernelThread {
    execPath = strings.TrimSuffix(string(cmdline), "\n")
} else {
    logger.GetLogger().WithError(err).WithField("process", d.Name()).Warnf("reading process exe error")
}
```